src/object/srv_ec_aggregate.c

```diff
@@ -1646,10 +1665,10 @@ agg_process_stripe(struct ec_agg_entry *entry)
 	iter_param.ip_hdl		= DAOS_HDL_INVAL;
 	iter_param.ip_ih		= entry->ae_thdl;
 	iter_param.ip_flags		= VOS_IT_RECX_VISIBLE;
+	iter_param.ip_recx.rx_nr	= ec_age2cs(entry);
 	iter_param.ip_recx.rx_idx	= PARITY_INDICATOR |
 					  (entry->ae_cur_stripe.as_stripenum *
-						entry->ae_oca->u.ec.e_len);
-	iter_param.ip_recx.rx_nr	= entry->ae_oca->u.ec.e_len;
+					   iter_param.ip_recx.rx_idx);
```

Comment:
L1671 seems typo/wrong? (should be iter_param.ip_recx.rx_nr, or ec_age2cs(entry))

Reply:
BTW, ec_age2cs(entry) looks better, although they are same.

Reply:
indeed, thanks